pkg/minishift/addon/parser/addon_parser_test.go

```diff
@@ -365,17 +280,10 @@ func Test_multiple_addon_definitions_create_error(t *testing.T) {
 	}
 
 	addon, err := testParser.Parse(testDir)
-
-	if err == nil {
-		t.Fatal("Expected a parse error for directory w/o addon file")
-	}
+	assert.Error(t, err)
 
 	expectedError := fmt.Sprintf(multipleAddOnDefinitionsError, strings.Join(addonFiles, ", "))
-	if err.Error() != expectedError {
-		t.Fatal(fmt.Sprintf("Unexpected error message. Got '%s'. Expected '%s'", err.Error(), expectedError))
-	}
+	assert.EqualError(t, err, expectedError)
 
-	if addon != nil {
-		t.Fatal("No addon should have been returned")
-	}
+	assert.Nil(t, addon)
```

Comment:
I know I repeat myself, but why not `assert.Nil(t, addon, "No addon should have been returned")`. This additional context does help. Especially once tests start failing. This applies to most asserts.